rma/tests/test_rma.py

```diff
@@ -155,6 +156,28 @@ def test_post_init_hook(self):
         self.assertTrue(warehouse.rma_in_route_id)
         self.assertTrue(warehouse.rma_out_route_id)
 
+    @mute_logger("odoo.models.unlink")
+    def test_migration_160140(self):
```

Comment:
I don't think this test is useful except for the migration scenario, which is not needed for future versions nor upstream changes, so I would remove it.